bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java

```diff
@@ -143,9 +149,9 @@ public void initialize() {
      * @param refresh : refresh frequency in minutes
      */
     private void startAutomaticRefresh(int refresh) {
-        if (refreshJob == null || refreshJob.isCancelled()) {
-            refreshJob = scheduler.scheduleWithFixedDelay(this::queryApiAndUpdateChannels, 10, refresh,
-                    TimeUnit.MINUTES);
+        if (!refreshJob.isPresent() || refreshJob.get().isCancelled()) {
```

Comment:
We always use `@Nullable` for the jobs in all bindings. I think it would be better to stay with that.

Reply:
Reverted